ElrondNetwork/api.elrond.com
# NFT update attributes log handler

## Type

## Problem setting
- NFT update attributes was handled only at transaction level and not smart contract result level
  
## Proposed Changes
- Use logs & events handler to update NFT attributes & refresh metadata

## How to test
- Update NFT attributes with a new metadata and see them reflected in the NFT details
